tensorboard/components/vz_example_viewer/vz-example-viewer.html

```diff
@@ -101,54 +102,60 @@
 
       .value-pills-holder {
         flex-grow: 1;
+        flex-basis: 100%;
+      }
+
+      iron-autogrow-textarea {
+        --iron-autogrow-textarea: {
+          font-size: 14px;
+          color: #3C4043;
+        }
+      }
+
+      iron-autogrow-textarea.value-same {
+        --iron-autogrow-textarea: {
+          font-size: 14px;
```

Comment:
You probably can omit font-size declaration in this and below cases.

Reply:
it didn't inherit from the more general rule when the font-size was in the mixin. i moved it up to the main part of the  iron-autogrow-textarea rule and now it inherits correctly and i removed it from the class-specific rules.